discord/commands.py

```diff
@@ -202,15 +205,18 @@ def target_channel(self, value: Optional[Messageable]) -> None:
         self._channel = value
 
 
-class SlashMixin(ApplicationCommand):
+class SlashMixin(ApplicationCommand, Protocol):
     if TYPE_CHECKING:
         _parent: SlashCommand
         options: List[Option]
         children: List[SubCommand]
 
     async def __call__(self, options, channel=None):
         obj = self._parent
+        command = obj._data
+        command['name_localized'] = command['name']
```

Comment:
What's the point of this line? Is an alias being added to `name`?

Reply:
Command localization is coming, but clients don't have access to it yet. They send this value anyway. When localization is released, this'll probably need to be changed.